packages/ui5-tooling-modules/test/__fixtures__/@ui5/webcomponents-fiori/classes/NotificationListGroupItem.json

```diff
@@ -1 +1 @@
-{"namespace":"@ui5/webcomponents-fiori","tag":"ui5-li-notification-group","interfaces":[],"properties":{"collapsed":{"type":"boolean","mapping":"property","defaultValue":false},"growing":{"type":"@ui5/webcomponents.NotificationListGrowingMode","mapping":"property","defaultValue":"None"},"titleText":{"type":"string","mapping":"property"},"read":{"type":"boolean","mapping":"property","defaultValue":false},"loading":{"type":"boolean","mapping":"property","defaultValue":false},"loadingDelay":{"type":"float","mapping":"property","defaultValue":1000},"text":{"type":"string","mapping":"textContent"},"width":{"type":"sap.ui.core.CSSSize","mapping":"style"},"height":{"type":"sap.ui.core.CSSSize","mapping":"style"}},"aggregations":{"items":{"type":"@ui5/webcomponents-fiori.NotificationListItem","multiple":true}},"associations":{},"events":{"toggle":{},"loadMore":{}},"getters":[],"methods":[],"defaultAggregation":"items"}
```

Comment:
This is the important part, the `text`, `width` and `height` property vanish for a couple of classes in case it is inherited from a superclass.